protocol/storage/miner_test.go

```diff
@@ -52,7 +53,7 @@ func TestReceiveStorageProposal(t *testing.T) {
 		}
 
 		vouchers := testPaymentVouchers(porcelainAPI, VoucherInterval, defaultAmountInc)
-		proposal := testSignedDealProposal(porcelainAPI, vouchers, porcelainAPI.targetAddress)
+		proposal := testSignedDealProposal(porcelainAPI, vouchers, 1000)
```

Comment:
Nit: It would be good to add a `defaultSize` constant in this file rather than all these magic numbers.